assets/js/components/TourTooltip.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * TourTooltip component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Card, { CardActions } from '@material/react-card';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CloseIcon from '../../svg/close.svg';
+import Button from './Button';
+
+/**
+ * Creates an array of length equal to size with incremental values starting at 0.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} size Array length.
+ * @return {number[]} Array of incremental numbers.
+ * @example createIncrementalArrayBySize(3); // [0, 1, 2]
+ */
+const createIncrementalArrayBySize = ( size ) => new Array( size ).fill().map( ( _, i ) => i );
+
+export default function TourTooltip( {
+	backProps,
+	closeProps,
+	index,
+	isLastStep,
+	primaryProps,
+	size,
+	step,
+	tooltipProps,
+} ) {
+	const indicatorArray = createIncrementalArrayBySize( size );
+	const getIndicatorClassName = ( indicatorIndex ) => classnames( 'googlesitekit-tooltip-indicator', {
+		active: indicatorIndex === index,
+	} );
+
+	const gotItText = __( 'Got it', 'google-site-kit' );
+
+	return (
+		<div className="googlesitekit-tour-tooltip" { ...tooltipProps }>
+			<Card className="googlesitekit-tooltip-card">
+				<div className="googlesitekit-tooltip-body">
+					<h2 className="googlesitekit-tooltip-title">
+						{ step.title }
+					</h2>
+					<div className="googlesitekit-tooltip-content">
+						{ step.content }
+					</div>
+				</div>
+				<CardActions className="googlesitekit-tooltip-actions">
+					<ul className="googlesitekit-tooltip-indicators">
+						{ indicatorArray.map( ( indicatorIndex ) => (
+							<li
+								key={ `indicator-${ indicatorIndex }` }
+								className={ getIndicatorClassName( indicatorIndex ) }
+							/>
+						) ) }
+					</ul>
```

Comment:
Currently, if there is only one step in the tour, it shows a single bullet which looks a bit confusing. What if we hide these indicators if we have less than 2 steps? cc @felixarntz

Reply:
Hiding indicators when tour only contains a single step seems like the most-clear option. I added this, but can quickly remove it if necessary.